### PR TITLE
interfaces/apparmor: switch to Revision

### PR DESCRIPTION
**Pull request by zyga**
This patch switches the generated apparmor policy to use snap revision
instead of snap version.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

**Comment by niemeyer** [comment]
LGTM, please wait for @pedronis.

File: interfaces/apparmor/template_vars.go [hunk 1]
```diff
@@ -47,15 +47,15 @@ func legacyVariables(appInfo *snap.AppInfo) []byte {
...
+		dbus.SafePath(fmt.Sprintf("%s.%s_%s_%d",
+			appInfo.Snap.Name(), appInfo.Snap.Developer, appInfo.Name, appInfo.Snap.Revision)))
 	// XXX: How is this different from APP_ID_DBUS?
 	fmt.Fprintf(&buf, "@{APP_PKGNAME_DBUS}=\"%s\"\n",
 		dbus.SafePath(fmt.Sprintf("%s.%s", appInfo.Snap.Name(), appInfo.Snap.Developer)))
 	// TODO: stop using .Developer, investigate how this is used.
 	fmt.Fprintf(&buf, "@{APP_PKGNAME}=\"%s\"\n", appInfo.Snap.Name())
 	// TODO: switch to .Revision
-	fmt.Fprintf(&buf, "@{APP_VERSION}=\"%s\"\n", appInfo.Snap.Version)
+	fmt.Fprintf(&buf, "@{APP_VERSION}=\"%d\"\n", appInfo.Snap.Revision)
```
Comment (pedronis) [inline interfaces/apparmor/template_vars.go]: I don't know if this is what we want or we want APP_REVISION? (it awkward) how is this used? maybe that's a question jdstrand

Comment (zyga) [inline interfaces/apparmor/template_vars.go]: We want ``SNAP_REVISION`` but I'd like to do that later (next week) as this is internal to apparmor.

Comment (pedronis) [inline interfaces/apparmor/template_vars.go]: ok, can you change the TODO then? then it can go in

Comment (zyga) [inline interfaces/apparmor/template_vars.go]: Sure

**Pull request merged**